datashuttle/datashuttle_class.py

```diff
@@ -868,10 +868,36 @@ def setup_ssh_connection(self) -> None:
 
         if verified:
             ssh.setup_ssh_key(self.cfg, log=True)
+            # 449
+            confirmed_path = (
```

Comment:
This is a nice idea, but taking an input here would freeze the application when run through the TUI. At this stage we can trust the user has input the path correctly at the config stage (if they have not, they will realize after a permission error below). Here I would delete the `input` code and use `self.cfg["central_path"]` directly in `self._check_write_permissions`